content/applications/inventory_and_mrp/manufacturing/management/split_merge.rst

```diff
@@ -0,0 +1,82 @@
+====================================
+Split and merge manufacturing orders
+====================================
+
+In Odoo *Manufacturing*, it is possible to create manufacturing orders for a single unit of an item
+or multiple units of the same item. In some cases, it may be necessary to split a manufacturing
+order that contains multiple units into two or more orders, or to merge two or more orders into a
+single order.
+
+.. important::
+   A manufacturing order can only contain one unit of a product or multiple units of a single
+   product that all use the same Bill of Materials (BoM). As a result, it is only possible to merge
+   manufacturing orders when every order contains the same product being manufactured with the same
+   BoM.
+
+Split manufacturing orders
+==========================
+
+To split a manufacturing order into multiple orders, begin by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, then select a manufacturing
+order. At the top of the page, next to the :guilabel:`New` button, the manufacturing order's
+reference number appears with a :guilabel:`⚙️ (settings)` button next to it.
+
+Click the :guilabel:`⚙️ (settings)` button to open the general settings for the manufacturing order,
+then select :guilabel:`Split`.
+
+.. image:: split_merge/settings-split.png
+   :align: center
+   :alt: The Settings and Split buttons on a manufacturing order.
+
+After selecting :guilabel:`Split`, a :guilabel:`Split production` pop-up window appears. In the
+:guilabel:`Split #` field, enter the number of manufacturing orders that the original order should
+be split into, then click outside of the field. A table appears below, with a line for each new
+manufacturing order that will be created by the split. In the :guilabel:`Quantity To Produce`
+column, enter the number of units that will be assigned to each new manufacturing order. Finally,
+click :guilabel:`Split` to split the manufacturing order.
+
+.. image:: split_merge/split-production-window.png
+   :align: center
+   :alt: The Split production pop-up window for a manufacturing order.
+
+After clicking :guilabel:`Split`, the original manufacturing order is split into the number of
+orders that was specified in the :guilabel:`Split #` field. The reference numbers for the new
+manufacturing orders are the reference number for the original order with *-XXX* tags added to the
```

Comment:
```suggestion
manufacturing orders are the reference number for the original order with a three-digit number starting at 1 added to the
```
Your preference as to whether XXX or "3-digit number" would make more sense to the average reader (: